src/main/java/roomescape/service/ReservationTimeService.java

```diff
@@ -0,0 +1,79 @@
+package roomescape.service;
+
+import org.springframework.stereotype.Service;
+import roomescape.domain.ReservationTime;
+import roomescape.domain.Theme;
+import roomescape.dto.time.ReservationTimeRequest;
+import roomescape.dto.time.ReservationTimeResponse;
+import roomescape.dto.time.available.ReservationTimeAvailableResponse;
+import roomescape.dto.time.create.ReservationTimeCreateResponse;
+import roomescape.exception.ErrorCode;
+import roomescape.exception.custom.DuplicatedReservationTimeException;
+import roomescape.exception.custom.ThemeNotFoundException;
+import roomescape.repository.ReservationRepository;
+import roomescape.repository.ReservationTimeRepository;
+import roomescape.repository.ThemeRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class ReservationTimeService {
+
+    private final ReservationRepository reservationRepository;
+    private final ReservationTimeRepository reservationTimeRepository;
+    private final ThemeRepository themeRepository;
+
+    public ReservationTimeService(ReservationRepository reservationRepository,
+                                  ReservationTimeRepository reservationTimeRepository,
+                                  ThemeRepository themeRepository) {
+        this.reservationRepository = reservationRepository;
+        this.reservationTimeRepository = reservationTimeRepository;
+        this.themeRepository = themeRepository;
+    }
+
+    public List<ReservationTimeResponse> findTimes() {
+        return reservationTimeRepository.findTimes()
+                .stream()
+                .map(ReservationTimeResponse::new)
+                .collect(Collectors.toList());
+    }
+
+    public ReservationTimeCreateResponse createTime(ReservationTimeRequest request) {
+        checkDuplicatedReservationTime(request.getStartAt());
+        ReservationTime reservationTime = ReservationTime.from(request);
+        ReservationTime response = reservationTimeRepository.createTime(reservationTime);
```

Comment:
```suggestion
        ReservationTime reservationTime = reservationTimeRepository.createTime(reservationTime);
```
해당 변수가 response 는 아니니 도메인과 같은 변수명을 사용해주면 좋을 것 같아요 ㅎㅎ